### ISSUE TITLE
Feature Request: add MonoPropellant to Karry Kan

### ISSUE THREAD

**Issue by enceos** [BobPalmer/Karbonite#136]
There are things around in KSP, powerred by RCS fuel. It would be nice if Kerbals could refuel them with a MonoProp Karry Kan.

**Comment by BobPalmer** [comment]
Query:  Given there is a KAS-able small RCS tank, what is the value add for the Karry Kan?

**Comment by enceos** [comment]
Considering the MFT ratios I'd add 20 monoprop to the small Karry Kan and 40 to the double.

**Comment by BobPalmer** [comment]
Let me rephrase :)

Why do you want monoprop in a karry kan when we already have a small portable RCS tank?

**Comment by enceos** [comment]
In Karbonite? What's the name of it?

**Comment by BobPalmer** [comment]
Bear in mind repaint is being taken out as it's kinda cheaty and a FireSpitter bug.

**Comment by enceos** [comment]
Ramarren has fixed it in the latest Firespitter.dll release, 'Repaint' now empties the containers after changing resources.

**Comment by BobPalmer** [comment]
Ok hopefully Sjno puts that in, in which case it opens up some nice opportunities
